platform/linux_platform.go

```diff
@@ -1380,7 +1380,7 @@ func (p linux) MigratePersistentDisk(fromMountPoint, toMountPoint string) error
 
 	// Golang does not implement a file copy that would allow us to preserve dates...
 	// So we have to shell out to tar to perform the copy instead of delegating to the FileSystem
-	tarCopy := fmt.Sprintf("(tar -C %s --xattrs -cf - .) | (tar -C %s --xattrs -xpf -)", fromMountPoint, toMountPoint)
+	tarCopy := fmt.Sprintf("(tar -C %s --xattrs --xattrs-include=*.* -cf - .) | (tar -C %s --xattrs --xattrs-include=*.* -xpf -)", fromMountPoint, toMountPoint)
```

Comment:
When you specify `*.*` you will be including the extended attributes of all the files that contain a dot in the name. I think this will not include files without a dot, for example: `Makefile` or `LICENSE`. Is the intention to leave this files out? If not, it might be better to use `--xattrs-include=*`

Reply:
`--xattrs-include` is not referring to filenames, as I understood [this](https://www.gnu.org/software/tar/manual/html_node/Extended-File-Attributes.html).
```
Here, the pattern is a globbing pattern. For example, the following command:
$ tar --xattrs --xattrs-exclude='user.*' -c a.tar .
will include in the archive ‘a.tar’ all attributes, except those from the ‘user’ namespace.

...

Attribute names are strings prefixed by a namespace name and a dot. Currently, four namespaces exist: ‘user’, ‘trusted’, ‘security’ and ‘system’.
```

This is why I think `*.*` should be correct. I even tested it with a file named `nginx-ingress-controller` which doesn't include a dot.

Reply:
thanks for the clarification.